### DISCUSSION TITLE
❓Getting Training Loss and Accuracy Logs for EfficientAD Anomaly Detection Model

### DISCUSSION THREAD

**Discussion by sasikaran20** [open-edge-platform/anomalib#3184] [Q&A]
### Your Question

i have using the efficientAD model for anomaly detection here in training is it possible to get the train loss and train accuracy for models in log or do you have any docs for that 

### Forum Check

- [x] I have searched the discussions forum for an answer to my question.

**Comment by github-actions** [comment]
Thanks for submitting this issue! It has been added to our triage queue. A maintainer will review it shortly.

**Comment by rajeshgangireddy** [comment]
Hi @sasikaran20 
The training loss is logged after every epoch. 
For EfficientAd, individual component losses and the final loss are logged. See [training_step code](https://github.com/open-edge-platform/anomalib/blob/74ade818f5e1cfbebdb35a04ffa1d0d8acdaaf35/src/anomalib/models/image/efficient_ad/lightning_model.py#L418)

If you want to log/store the loss after each epoch, you can also use one of the other loggers supported by Anomalib. 
See https://anomalib.readthedocs.io/en/latest/markdown/guides/reference/loggers/index.html for details.
